l10n_it_reverse_charge/tests/test_rc.py

```diff
@@ -52,6 +64,11 @@ def _create_taxes(self):
             'type_tax_use': 'sale',
             'amount': 22
         })
+        self.tax_21 = tax_model.create({
+            'name': "Tax 21%",
+            'type_tax_use': 'purchase',
+            'amount': 21
```

Comment:
@SimoRubi 21 non è molto reale come aliquota

Reply:
Fatto in https://github.com/OCA/l10n-italy/pull/561/commits/147d251a7918b2b7602eb5b7a58b80341e8019db